### DISCUSSION TITLE
Some .js files not served on github.io

### DISCUSSION THREAD

**Discussion by tliawi** [community/community#23981] [Other Feature Feedback, Questions, & Ideas]
<p><a href="http://tliawi.github.io/BMHCmap/" rel="noopener nofollow ugc">tliawi.github.io/BMHCmap/</a>  has an index.html that uses three script src files: bmhc.js, bmhcTags.js, and bmhcData.js. It displays but is nonfunctional because the first and the last of those .js files are 404’d (you can see it in the javascript console)–all the rest of the files (a css, bmhcTags.js, other .js files in subdirectories) are available. For example, <a href="https://tliawi.github.io/BMHCmap/bmhcTags.js" rel="noopener nofollow ugc">https://tliawi.github.io/BMHCmap/bmhcTags.js</a> serves it up fine, but <a href="https://tliawi.github.io/BMHCmap/bmhc.js" rel="noopener nofollow ugc">https://tliawi.github.io/BMHCmap/bmhc.js</a> is 404. I’ve tried (in my project settings for pages) shutting down pages, waiting until indeed <a href="http://tliawi.github.io/BMHCmap/" rel="noopener nofollow ugc">tliawi.github.io/BMHCmap/</a> was inoperative, and then starting it up again. Everything is default. Nothing works, including waiting 8 hours.</p>

**Comment by tliawi** [comment, marked as answer]
<p>Never mind. Crazy, those files’ names were in lowercase in my macOS file system, but for some reason my local gitHub and the web gitHub both had them with uppercase filenames. Maybe they started out that way, and I changed them somewhere along the line many commits ago, but gitHub didn’t pick up on it. I fixed it by appending xxx to the (lowercase) names on my file system, committing, removing the xxx from those names, and committing again, so that gitHub would pick up on the change in case. Voila, now everybody thinks they’re in lower case, and everything works. Cost me a day. Dammit, this business of “GitHub DOES NOT SEE CHANGES IN FILE NAMES IF THEY ONLY INVOLVE CHANGES IN CASE” should be documented on every street corner. It bites. Nothing in my vain searches for “<a href="http://github.io" rel="noopener nofollow ugc">github.io</a> javascript src 404” crossed paths with the issue.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I’m sorry you had this trouble, and delighted you figured it out. Thank you for leaving it all here too - chances are you will save someone else in our community from the same heartache.</p>
